src/App/Fossa/Config/Analyze.hs

```diff
@@ -255,42 +253,59 @@ instance ToJSON ExperimentalAnalyzeConfig where
   toEncoding = genericToEncoding defaultOptions
 
 mkSubCommand :: (AnalyzeConfig -> EffStack ()) -> SubCommand AnalyzeCliOpts AnalyzeConfig
-mkSubCommand = SubCommand "analyze" analyzeInfo cliParser loadConfig mergeOpts
+mkSubCommand = SubCommand ("analyze") analyzeInfo cliParser loadConfig mergeOpts
 
 analyzeInfo :: InfoMod a
-analyzeInfo = progDesc "Scan for projects and their dependencies"
+analyzeInfo = progDescDoc (formatStringToDoc "Scan for projects and their dependencies")
```

Comment:
[Prefer] Include example usage - maybe something like following? Since the output is quite large, it might be easier for users just to copy paste commands. As a new user if I see these many options,  I might be taken back on need to choose right options. I think we should provide example for common usages.

We should prob. do this for all commands.

```
analyzeInfo :: InfoMod a
analyzeInfo = progDescDoc . Just $ vsep [ -- should prob color these as well
    "Scan for projects and their dependencies",
    " ",
    vsep [
      "Examples:",
      "---------",
      vsep [
        "- fossa analyze .   # analyze current directory recursively",
        "- fossa analyze src # analyze specific directory recursively",
        "- fossa analyze . --detect-vendored # analyze and detect vendored open source libraries",
        "- fossa analyze . --debug"
      ]
    ]
  ]
```